src/drawing/DrawingPath.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { v4 as uuidv4 } from 'uuid';
+import { white } from 'box-ui-elements/es/styles/variables';
+import { DrawingSVGRef } from './DrawingSVG';
+import { PathGroup } from '../@types';
+import { Point } from '../region/transformUtil';
+import { isVectorEffectSupported } from './drawingUtil';
+import './DrawingPath.scss';
+
+export type Props = {
+    isActive?: boolean;
+    pathGroup: PathGroup;
+    rootEl: DrawingSVGRef | null;
+};
+
+export const DRAWING_TARGET_BORDER = 1;
+
+export const getPathCommands = (points: Point[]): string => {
+    if (!points || !points.length) {
+        return '';
+    }
+
+    const { x: startX, y: startY } = points[0];
+    const d = points
+        .map(({ x, y }, index, array) => {
+            const prevPoint = array[index - 1];
+            const nextPoint = array[index + 1];
+            if (!prevPoint || !nextPoint) {
+                return '';
+            }
+
+            const xc1 = (x + prevPoint.x) / 2;
+            const yc1 = (y + prevPoint.y) / 2;
+            const xc2 = (x + nextPoint.x) / 2;
+            const yc2 = (y + nextPoint.y) / 2;
+
+            return `C ${xc1} ${yc1}, ${x} ${y}, ${xc2} ${yc2}`;
+        })
+        .join(' ');
+
+    return `M ${startX} ${startY} ${d}`;
+};
+
+export const DrawingPath = (props: Props): JSX.Element => {
+    const {
+        isActive = false,
+        pathGroup: {
+            paths,
+            stroke: { color, size },
+        },
+        rootEl,
+    } = props;
+
+    let strokeWidth = size;
+    let strokeBorderWidth = size + DRAWING_TARGET_BORDER * 2;
+
+    if (!isVectorEffectSupported()) {
+        if (!rootEl) {
+            return <g />;
+        }
+        const { clientHeight, clientWidth } = rootEl;
+        const scale = Math.min(clientHeight, clientWidth) / 100;
+        strokeWidth /= scale;
+        strokeBorderWidth /= scale;
+    }
+
+    return (
+        <g className="ba-DrawingPath" fill="transparent" stroke={color} strokeWidth={strokeWidth}>
+            {paths.map(({ points }) => {
+                const pathCommands = getPathCommands(points);
+                return (
+                    <g key={uuidv4()}>
+                        <g className={classNames('ba-DrawingPath-decoration', { 'is-active': isActive })}>
```

Comment:
Could `is-active` bet set on the topmost `g` element?

Reply:
sure, I'll try